MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -32,6 +82,26 @@ class RouteManeuverViewController: UIViewController {
         }
     }
     
+    var availableStreetLabelBounds: CGRect {
+        return CGRect(origin: .zero, size: maximumAvailableStreetLabelSize)
```

Comment:
AbridgedLabel could inherit from various style labels so that wouldn't be convenient.
Instead I added `Abridgeable` protocol that any label can conform to.

I wanted to use `class AbridgedLabel<T:StyleLabel> : Abridgeable { }` but generics can't be hooked up to an IBOutlet. 😔